contribs/gnofaucet/serve.go

```diff
@@ -55,26 +56,29 @@ type serveCfg struct {
 	maxSendAmount string
 	numAccounts   uint64
 
-	remote string
-
-	captchaSecret string
+	remote        string
 	isBehindProxy bool
 }
 
 func newServeCmd() *commands.Command {
 	cfg := &serveCfg{}
-
-	return commands.NewCommand(
+	cmd := commands.NewCommand(
 		commands.Metadata{
 			Name:       "serve",
-			ShortUsage: "serve [flags]",
+			ShortUsage: "<subcommand> [flags]",
+			ShortHelp:  "serve <subcommand> [flags]",
 			LongHelp:   "Serves the gno.land faucet to users",
 		},
 		cfg,
-		func(ctx context.Context, args []string) error {
-			return execServe(ctx, cfg, commands.NewDefaultIO())
-		},
+		commands.HelpExec,
 	)
+
+	cmd.AddSubCommands(
```

Comment:
Can you add a README.md that explains how to use the captcha and how to use the GH app?

Reply:
What do you think of
a4b8827237053be578aee85e87c92d26fe2239da ?